spec/Section 3 -- Type System.md

```diff
@@ -1144,9 +1146,9 @@ interface. Querying for `age` is only valid when the result of `entity` is a
 **Interfaces Implementing Interfaces**
 
 When defining an interface that implements another interface, the implementing
-interface must define each field that is specified by the implemented interface.
-For example, the interface Resource must define the field id to implement the
-Node interface:
+type must define each field that is specified by the implemented interface. For
```

Comment:
I don't think this needs to change here, as in the context, we are specifically referring to interfaces implementing interfaces, so the "implementing type" will be an "implementing interface.